### DISCUSSION TITLE
Require/encourage alt text for asset uploads

### DISCUSSION THREAD

**Discussion by turnstylerj** [craftcms/cms#14028] [Ideas]
No matter how much training we do, when we hand over a build to a client, they immediately do two things:
- Upload images way too large (since we can't set size limits for individual volumes)
- Stop adding alt text to new image uploads

For alt text, it would be nice if there were a way to encourage the filling out of alt text because it's a lot to ask them to select an image and then remember to double-click on its chip to add alt text in the overlay. 

I realize this is a tricky matter; my initial thought would be an option for each asset volume that would open a newly-uploaded asset's entry view in an overlay as soon as it's uploaded, so the author would need to add alt text (if we made it a required field), and save before actually using the image.

**Comment by wmdhosting** [comment]
There is  resize plugin.. on upload..

As for Alt text. you can doit in way if they do not put alt text..  use Entry title as ALt text.

**Reply by turnstylerj** [reply]
Good to know about Image Resizer plugin! We've used asset entry title for alt text in the past, but it just resulted in a lot of unhelpful alt text like "DCIM_3245". So we'd really like the actual alt text field with instructions to be more prominent/actually required.

**Comment by wmdhosting** [comment]
i mean entry title..  not asset title

**Reply by turnstylerj** [reply]
Alt text is supposed to be a description of an image for someone who can't see it, not the title of the entry.

**Comment by wmdhosting** [comment]
i understand. but its better  entry title than nothing in most cases...
But ok you know better what you need
